maro/simulator/scenarios/vm_scheduling/cluster.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import List
+
+from maro.backends.frame import NodeAttribute, NodeBase, node
+
+
+@node("clusters")
+class Cluster(NodeBase):
+    """Cluster node definition in frame."""
+    id = NodeAttribute("i2")
+    region_id = NodeAttribute("i2")
+    zone_id = NodeAttribute("i2")
+    data_center_id = NodeAttribute("i2")
+    # The number of empty machines in this cluster. A empty machine means that its allocated CPU cores are 0.
+    empty_machine_num = NodeAttribute("i")
+
+    def __init__(self):
+        self._id: int = 0
+
+        self._cluster_type: str = ""
+        self._rack_list: List[int] = []
+
+    def set_init_state(self, id: int):
+        """Set initialize state, that will be used after frame reset.
+
+        Args:
+            id (int): Region id.
+        """
+        self._id = id
+
+        self.reset()
+
+    def reset(self):
+        """Reset to default value."""
+        self.id = self._id
+        self._cluster_type = ""
+        self._rack_list.clear()
+
+        self.region_id = -1
```

Comment:
All these id and type will keep their previous values when BE.reset()?

Are you setting these to -1 here and reset them outside this class?

Reply:
Similar comments to class region/zone/data_center/rack/...